algorithms/sorting.md

```diff
@@ -32,18 +32,18 @@ For example bubble sort, quick sort, and selection sort all take the same input
  
 ### Pre-Lesson Warm Up and Game (30 minutes)
 - Discussion topic: What is an algorithm?  Can you give us an example of an algorithm?
-  - An algorithm is a way for describing automated processes
+  - An algorithm is a way for describing automated processes.
 - Discussion topic: What is a Sorting Algorithm? Can you give us an example of something that we sort in our day-to-day lives?
-  - A Sorting Algorithm orders items in a group according to a specificed attribute. ie: Order a roster of students by name, Order cartons of milk by expiration date
+  - A Sorting Algorithm orders items in a group according to a specificed attribute. ie: Order a roster of students by name, Order cartons of milk by expiration date.
 - Discussion topic: Why do we need sorting algorithms in Computer Science?
-  - Let's do a Card-Sorting activity. Partner up in groups of two or three, and order cards from lowest to highest (all of the suite). Give student groups a deck of cards and give them each a suite of cards to sort.
+  - Let's do a Card-Sorting activity. Partner up in groups of two or three and order cards from lowest to highest (all of the same suit). Give student groups a deck of cards and give them each a suit of cards to sort.
   - After 10 minutes of sorting, let's reflect on how we accomplished it. Did we sort the cards differently from the people in our partner groups?  Did we compare a few at a time?  Did we look for the lowest first and then go from there?  Do you think we could sort in fewer "moves"?
-  - After this Card-Sorting activity, it's clear there are a lot of different ways to sort cards!  This translates right over to how computer scientists sort Lists. As Computer Scientists, we have lots of different algorithms to use that allow us to sort - from BubbleSort, to MergeSort, to QuickSort!  Some algorithms aren't as fast as others (Obama knows not to use the BubbleSort: [Watch video](https://www.youtube.com/watch?v=k4RRi_ntQc8)), but there are many tradeoffs to different algorithms!
+  - After this Card-Sorting activity it's clear there are a lot of different ways to sort cards!  This translates right to how computer scientists sort Lists. As Computer Scientists we have lots of different algorithms to use that allow us to sort - from Bubble Sort, to Merge Sort, to Quick Sort!  Some algorithms aren't as fast as others (Obama knows not to use the Bubble Sort: [Watch video](https://www.youtube.com/watch?v=k4RRi_ntQc8)), but there are many tradeoffs to different algorithms!
```

Comment:
Thank you for the revisions.  Are you changing BubbleSort, etc. to Bubble Sort because it is the correct CS term, or is this from a spellcheck  program?